src/test/java/hw6/Test/JdiTest.java

```diff
@@ -0,0 +1,48 @@
+package hw6.Test;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import hw6.DataProviderJsonForMetalsColorPage;
+import hw6.JdiSite;
+import hw6.Json.JsonDataProvider;
+import hw6.complexelements.HeaderMenuItems;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+@Test(dataProvider = "DataProviderJson", dataProviderClass = JsonDataProvider.class)
+public class JdiTest {
+
+    @BeforeSuite
+    public void beforeSuite() {
+        PageFactory.initSite(JdiSite.class);
+    }
+
+    @AfterSuite
+    public void afterSuite() {
+        WebDriverUtils.killAllSeleniumDrivers();
+    }
+
+    @Test
+    public void jdiTest(DataProviderJsonForMetalsColorPage dataProviderJsonForMetalsColorPage) {
+
+        JdiSteps steps = new JdiSteps();
```

Comment:
We haven't used this approach in classwork, but OK. Anyway, this initialisation should not be inside of the test method.